### ISSUE TITLE
[Question] Can you set syntax highlighting only in editor window of Markdown?

### ISSUE THREAD

**Issue by Dapapap** [BoostIO/Boostnote#2401]
I've been looking around and I just can't find any mention of this. 
I'm trying to use Boostnote for some math notes (LaTeX support is a godsend), but when i type in LaTeX, it's a bit hard to follow when I look back on it. I am still pretty new to coding and Boostnote in general, and syntax highlighting always helped a ton. I found one solution that helps in editor, but completely defeats the purpose of LaTeX in preview mode.
# Current behavior
![markdown](https://user-images.githubusercontent.com/40748219/45588552-88aea000-b8e4-11e8-818f-3600e79033c9.png)
# Expected behavior
I would like the syntax highlighting to stay in the editor window only, but to render correctly in LaTeX for preview mode.

# Environment

- Version : 0.11.9
- OS Version and name : Windows 10 Pro

**Comment by ZeroX-DG** [comment]
Currently there's no way to achieve this feature. The best way to do it now is to wrap it in a codeblock like that to have the syntax highligting and remove the codeblock mark after that.
